cmd/operator-sdk/generate/packagemanifests/packagemanifests.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packagemanifests
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	genutil "github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate/internal"
+	gencsv "github.com/operator-framework/operator-sdk/internal/generate/clusterserviceversion"
+	"github.com/operator-framework/operator-sdk/internal/generate/collector"
+	genpkg "github.com/operator-framework/operator-sdk/internal/generate/packagemanifest"
+	"github.com/operator-framework/operator-sdk/internal/scaffold/kustomize"
+)
+
+//nolint:lll
+const examples = `
+	# Generate manifests then create the package manifests base:
```

Comment:
Align spacing with the rest of the example.
```suggestion
  # Generate manifests then create the package manifests base:
```